drf_to_mkdoc/utils/commons/operation_utils.py

```
@@ -0,0 +1,76 @@
+                mapping[operation_id] = path, action_data.get("parameters", [])
+
+    return mapping
+
+
+def extract_viewset_from_operation_id(operation_id: str):
+    """Extract the ViewSet class from an OpenAPI operation ID."""
+    operation_map = get_operation_id_path_map()
+    path, parameters = operation_map.get(operation_id)
+
+    if not path:
+        raise ValueError(f"Path not found for operation ID: {operation_id}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)